raiden/tests/integration/api/rest/test_rest.py

```diff
@@ -937,7 +939,10 @@ def test_channel_events_raiden(
 @pytest.mark.parametrize("number_of_nodes", [3])
 @pytest.mark.parametrize("channels_per_node", [CHAIN])
 @pytest.mark.parametrize("enable_rest_api", [True])
-def test_pending_transfers_endpoint(raiden_network: List[RaidenService], token_addresses):
+@patch("raiden.message_handler.decrypt_secret", side_effect=InvalidSecret)
```

Comment:
We usually use pytest's `monkeypatch` fixture instead of `patch` in the test suite (although not always...).
The advantage is that the monkeypatch fixture is integrated with pytest and also defaults to raising an exception if the patch target doesn't exist (e.g. after a refactoring) which is not the case with `patch`.

I think it would be nicer to put this into a fixture and / or test helper (e.g. `disable_encrypted_secret`) so as not having to repeat this patch boilerplate on many tests (which would need to be changed if ever the `decrypt_secret` function was moved or otherwise refactored).

Also this only tests the branch where the included secret is invalid, not the one where none is provided in the first place.

Reply:
Created https://github.com/raiden-network/raiden/issues/7287 for this